cmd/oras/internal/display/status/text.go

```diff
@@ -179,3 +181,106 @@ func (ch *TextCopyHandler) OnMounted(_ context.Context, desc ocispec.Descriptor)
+func NewTextManifestIndexCreateHandler(printer *output.Printer) ManifestIndexCreateHandler {
+	tmich := TextManifestIndexCreateHandler{
+		printer: printer,
+	}
+	return &tmich
+}
+
+// TextManifestIndexUpdateHandler handles text status output for manifest index update events.
+type TextManifestIndexUpdateHandler struct {
+	printer *output.Printer
+}
+
+// OnIndexFetching implements ManifestIndexUpdateHandler.
+func (miuh TextManifestIndexUpdateHandler) OnIndexFetching(indexRef string) error {
+	return miuh.printer.Println(IndexPromptFetching, indexRef)
+}
+
+// OnIndexFetched implements ManifestIndexUpdateHandler.
+func (miuh TextManifestIndexUpdateHandler) OnIndexFetched(indexRef string, digest digest.Digest) error {
+	if contentutil.IsDigest(indexRef) {
+		return miuh.printer.Println(IndexPromptFetched, indexRef)
+	}
+	return miuh.printer.Println(IndexPromptFetched, digest, indexRef)
+}
+
+// OnManifestFetching implements ManifestIndexUpdateHandler.
+func (miuh TextManifestIndexUpdateHandler) OnManifestFetching(ref string) error {
+	return miuh.printer.Println(IndexPromptFetching, ref)
+}
+
+// OnManifestFetched implements ManifestIndexUpdateHandler.
+func (miuh TextManifestIndexUpdateHandler) OnManifestFetched(ref string, digest digest.Digest) error {
+	if contentutil.IsDigest(ref) {
+		return miuh.printer.Println(IndexPromptFetched, ref)
+	}
+	return miuh.printer.Println(IndexPromptFetched, digest, ref)
+}
+
+// OnManifestRemoved implements ManifestIndexUpdateHandler.
+func (miuh TextManifestIndexUpdateHandler) OnManifestRemoved(digest digest.Digest) error {
+	return miuh.printer.Println(IndexPromptRemoved, digest)
+}
+
+// OnManifestAdded implements ManifestIndexUpdateHandler.
+func (miuh TextManifestIndexUpdateHandler) OnManifestAdded(ref string, digest digest.Digest) error {
+	if contentutil.IsDigest(ref) {
+		return miuh.printer.Println(IndexPromptAdded, ref)
+	}
+	return miuh.printer.Println(IndexPromptAdded, digest, ref)
+}
+
+// OnIndexMerged implements ManifestIndexUpdateHandler.
+func (miuh TextManifestIndexUpdateHandler) OnIndexMerged(ref string, digest digest.Digest) error {
+	if contentutil.IsDigest(ref) {
+		return miuh.printer.Println(IndexPromptMerged, ref)
+	}
+	return miuh.printer.Println(IndexPromptMerged, digest, ref)
+}
+
+// OnIndexUpdated implements ManifestIndexUpdateHandler.
+func (miuh TextManifestIndexUpdateHandler) OnIndexUpdated(digest digest.Digest) error {
+	return miuh.printer.Println(IndexPromptUpdated, digest)
+}
+
+// OnIndexPushed implements ManifestIndexUpdateHandler.
+func (miuh TextManifestIndexUpdateHandler) OnIndexPushed(indexRef string) error {
+	return miuh.printer.Println(IndexPromptPushed, indexRef)
+}
+
+// NewTextManifestIndexUpdateHandler returns a new handler for manifest index create command.
+func NewTextManifestIndexUpdateHandler(printer *output.Printer) ManifestIndexUpdateHandler {
+	miuh := TextManifestIndexUpdateHandler{
+		printer: printer,
+	}
+	return &miuh
```

Comment:
```suggestion
	manifestHandler := TextManifestIndexUpdateHandler{
		printer: printer,
	}
	return &manifestHandler
```

Reply:
The current naming style is aligned with the rest of the file.